server/server.js

```diff
@@ -211,55 +254,58 @@ app.post('/genotet/upload', upload.single('file'), function(req, res) {
     dataName: req.body.name,
     description: req.body.description
   };
-  uploader.uploadFile(body, req.file, dataPath, bigWigToWigPath,
+  uploader.uploadFile(body, req.file, dataPath, bigWigToWigPath, req.username,
     function(ret) {
       jsonResponse(/** @type {Object} */(ret), res);
     });
 });
 
 
-app.post('/genotet/user', function(req, res) {
-  log.serverLog('POST user');
+app.get('/genotet/user', function(req, res) {
```

Comment:
Since now get/post both works with credentials, maybe we can just keep POST here as well?